docs/quickstart/svelte-kit.mdx

```diff
@@ -121,28 +125,81 @@ If the files are valid, they will be automatically added to the `posts` collecti
 </div>
 <div className="step">
 
-Usage in your code:
+Usage in your code, e.g. as post list:
 
+Create in a `src/posts` directory
+
+`+page.ts`
 ```svelte
-<script>
```

Comment:
It seems that `svelte` is not the appropriate language; I believe `typescript` will resolve the broken syntax highlighting.

https://content-collections-git-fork-git-no-patch-1-nitilon.vercel.app/docs/quickstart/svelte-kit

Reply:
Completed